README.md

```diff
@@ -1,63 +1,55 @@
 # mros_reasoner
+
 A meta-controller implementation for ROS1
 
 ## Installation
 
 ### Previous steps
 
-- First you need to have a ROS Melodic version with Python3 support, for example following these [instructions](https://answers.ros.org/question/237613/how-to-define-ros-kinetic-to-use-python3-instead-of-python27/?answer=331009#post-id-331009)
-_Notes:_
-  - For me it worked with `Melodic` and `python3.6-venv`
-  - I created `$HOME/rospy3_melodic` for the ROS Melodic version with Python3 support
-
-- **Note:** For regular usage you'll need to make sure to have the virtual environment activated, or things will break.
+- you need to install a `java jre` you can install it with:
 
-- you need to install owlready2:
-```
-pip3 install owlready2
-```
-- You also need a `java jre` you can install it with:
-```
+```console
 sudo apt-get install openjdk-11-jre
 ```
+
 ### Create reasoner_ws
 
-- We recommend you now create a workspace only for `mros1_reasoner` (and any other packages requiring Python3 in your project), using [workspace overlaying](http://wiki.ros.org/catkin/Tutorials/workspace_overlaying), for example:
-```
-mkdir -p ~/mros1_reasoner_ws/src
-cd mros1_reasoner_ws
+- We recommend you to create a workspace only for `mros1_reasoner`, for example:
+
+```console
+  mkdir -p ~/mros1_reasoner_ws/src
+  cd mros1_reasoner_ws
 ```
 
 ### Get mros1_reasoner and dependencies using wstool
 
 - You need to copy the `mros1_reasoner` and its dependencies into the Python 3 reasoner workspace (ie: `mros1_reasoner_ws`), so they now exists in both the Python 2 as well as the Python 3 workspaces.
 
-```
-$ cd ~/mros1_reasoner_ws
-$ wstool init src https://raw.githubusercontent.com/rosin-project/mros1_reasoner/mvp/mros1_reasoner/mros1_reasoner.rosinstall
+```console
+  cd ~/mros1_reasoner_ws
+  wstool init src https://raw.githubusercontent.com/rosin-project/mros1_reasoner/mvp/mros1_reasoner/mros1_reasoner.rosinstall
+  rosdep install --from-paths ./src -y -i -r --skip-keys="abb_rws_interface"
```

Comment:
Might want to consider using absolute paths to the `src` dir, just in case users forget to `cd` earlier.

Reply:
And perhaps mention that `-r` will cause `rosdep` to ignore errors.

While this can be desirable, it will also mean users "get the prompt back" after this is run, and if they don't examine the output of `rosdep` carefully, might not spot problems with installing dependencies.